gimmemotifs/orthologs.py

```diff
@@ -249,10 +283,10 @@ def annot2primpep(genome, outdir):
     records = dict()
     for record in proteins:
         # get the gene name and gene id of the protein
-        prot_name = "|".join(record.long_name.split("\t")[1:])
+        prot_name = "|".join(re.split(r'[\s|~]+', record.long_name)[1:3])
```

Comment:
What is this for? In case you use your own annotation and want to use whitespace, bars or tildes?